motion_primitives_forward_controller/package.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>motion_primitives_forward_controller</name>
+  <version>0.0.0</version>
+  <description>Package to control robots using motion primitives like PTP, LIN and CIRC</description>
+  <maintainer email="todo@todo.de">todo</maintainer>
+  <author>Mathias Fuhrer</author>
```

Comment:
email for you could be defined, the release version could be updated to match the rest of the repo and please copy the maintainer fields from other controllers in t his repo